docs/reference/providers/backend.md

```diff
@@ -110,13 +111,36 @@ Google [Gemini](https://blog.google/technology/ai/google-gemini-ai/#performance)
 - To use Google Gemini API in K8sGPT, obtain [the API key](https://ai.google.dev/tutorials/setup).
 - To configure Google backend in K8sGPT with `gemini-pro` model (see all [models](https://ai.google.dev/models) here) use auth command:
     ```bash
-    k8sgpt auth add --backend google --model gemini-pro --password "<Your API KEY>"
+    k8sgpt auth add --backend googlevertexai --model gemini-pro --password "<Your API KEY>"
     ```
 - Run the following command to analyze issues within your cluster with the Google provider:
     ```bash
     k8sgpt analyze --explain --backend google
     ```
 
+## Google Gemini via Vertex AI
+
+Google [Gemini](https://blog.google/technology/ai/google-gemini-ai/#performance) allows generative AI capabilities with multimodal approach (it is capable to understand not only text, but also code, audio, image and video). 
+
+- To use [Google Vertex AI](https://cloud.google.com/vertex-ai?#build-with-gemini) you need to be authorized via [Google Cloud SDK](https://cloud.google.com/sdk/install). 
+    The [Vertex AI API](https://console.cloud.google.com/apis/library/vertex-ai.googleapis.com) needs to be enabled.
+
+> Note: Vertex AI Gemini API is currently available in these [regions](https://cloud.google.com/vertex-ai/docs/generative-ai/model-reference/gemini?hl=de#http_request)
```

Comment:
Is this expected to change? Maybe write something more along the lines of:
"Verify that the Vertex AI Gemini API is available in your [region](https://cloud.google.com/vertex-ai/docs/generative-ai/model-reference/gemini?hl=de#http_request)"

This makes it sound more timeless independent to how things change on the Google side :)

Reply:
Agree to some degree, the only issue I have is, you can basically use the other regions to debug clusters not in this region. I added a more clarification hope this fits!